src/fwrite.c

```diff
@@ -665,17 +671,20 @@ void fwriteMain(fwriteMainArgs args)
         STOP(_("Internal error: type %d has no max length method implemented"), args.whichFun[j]);  // # nocov
       }
     }
-    if (args.whichFun[j]==WF_Float64 && args.scipen>0) width+=MIN(args.scipen,350); // clamp width to IEEE754 max to avoid scipen=99999 allocating buffer larger than can ever be written
```

Comment:
`MIN` macro seems useful here for readability, any reason to remove it?

Reply:
https://lwn.net/Articles/983965 shows limit with MIN macro function. No problem if you restore MIN macro but I prefer direct test.

Reply:
Excellent reference, wow! I think our case falls under the "obvious const" heading, and we've stuck with trivially ignoring concerns about matching types etc. But I'm swayed it's better to avoid macros -- certainly better to avoid the temptation to bring on further complexity in the future by leaving it out now.